configs/annex.json

```diff
@@ -0,0 +1,42 @@
+{
+    "targets": [
+        {
+            "id": "main_db",
+            "type": "directory",
+            "path": "/home/montagu/starport/main_db_restore",
+            "s3_bucket": "montagu-db",
+            "encrypted": true
+        },
+        {
+            "id": "orderly",
+            "type": "directory",
+            "path": "/home/montagu/starport/orderly",
+            "s3_bucket": "montagu-orderly",
+            "encrypted": true
+        },
+        {
+            "id": "teamcity",
+            "type": "directory",
+            "path": "/home/montagu/starport/teamcity",
+            "s3_bucket": "montagu-teamcity",
+            "encrypted": true
+        },
+        {
+            "id": "vault",
+            "type": "directory",
+            "path": "/home/montagu/starport/vault",
+            "s3_bucket": "montagu-vault",
+            "encrypted": false
+        },
+        {
+            "id": "annex_db",
+            "type": "container",
+            "name": "montagu_db_annex",
+            "path": "/home/montagu/backups/annex_db.dump",
+            "backup_script": ["pg_dump", "--user=vimc", "-Fc", "--no-privileges", "montagu"],
```

Comment:
If this is going to run every night, this is going to be a pretty big drain on the machine. I wonder if we'd be better off separating the dump here from the duplicati. So backup a directory, same as the other tasks, and then run a weekly cron job that does the dump. Then we can remove all the before/after steps pretty much

Reply:
Okay. I'll just remove it from this PR, and raise a ticket for writing the weekly cron job.

Reply:
No wait, there's a problem with this - then we'll have two copies of the Annex on the annex machine, and there isn't room for that. In fact, that's a problem with both approaches.

I think the only reasonable way of backing up the Annex is to get Barman working properly in the cloud, and backing up continuously. That way we don't have huge uploads, and we can provision a big disk in the cloud.